releasenotes/notes/1.3/prepare-1.3.0-7c45598775fc2bbf.yaml

```
@@ -0,0 +1,22 @@
+---
+prelude: >
+    Qiskit version 1.3.0 brings in major performance and quality improvements
+    for the transpiler. The minimum required Python version is now 3.9.
+
+    There have been many new features, fixes and improvements introduced in this
+    new version of Qiskit, some of which include:
```

I'd say highlights or something to else to put more importance on this. These are the top features in this release.

```suggestion
    new version of Qiskit, the highlights are:
```